### PR TITLE
cmd/bnsd: multisig comments

### PR DESCRIPTION
**Pull request by ruseinov**
On request from @webmaster128. 
#trivial

### CODE REVIEW

File: cmd/bnsd/app/codec.proto [hunk 1]
```diff
@@ -30,6 +30,8 @@ message Tx {
   cash.FeeInfo fees = 1;
   repeated sigs.StdSignature signatures = 2;
   // ID of a multisig contract.
+  // nil array results in multisig being ignored, basically the same behaviour
```
Comment (webmaster128) [inline cmd/bnsd/app/codec.proto]: Is "nil array" the same as "empty array"? I think the later would be easier to understand outside of go world. Thanks!

Comment (ruseinov) [reply cmd/bnsd/app/codec.proto]: done

**Comment by ruseinov** [comment]
@webmaster128 good catch, done!

**Comment by iov-bot-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry. <br />
You can find it at <a href="https://github.com/iov-one/weave/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**